src/manager.js

```diff
@@ -0,0 +1,91 @@
+const Cohort = require('../src/cohort')
+const Student = require('../src/student')
+
+class Manager {
+  constructor() {
+    this.cohorts = []
+    this.studentID = 1
+  }
+
+  getCohort(cohortName) {
+    const cohort = this.cohorts.find((cohort) => cohort.name === cohortName)
+    if (cohort) return cohort
+    return false
+  }
+
+  createCohort(cohortName) {
+    if (!cohortName) return 'Cannot create a cohort without a name.'
+    const cohortExists = this.cohorts.find((el) => el.name === cohortName)
```

Comment:
I wonder what that `(el)` next to the `.find()` method does.

Reply:
el (short for element) represents the current element in the cohorts array. For each el, it checks if the name of el is equal to cohortName (the name passed into the method to create a cohort.